modules/test-controller-deploy/main.tf

```diff
@@ -538,6 +539,11 @@ BUILDSPEC
   tags = local.tags
 }
 
+resource "aws_codestarconnections_connection" "this" {
```

Comment:
Will require a login to codepipeline from the console
Fixes
> Fixes
Warning: The CodePipeline GitHub version 1 action provider is deprecated.